src/main/java/kr/co/wordle/support/WordFileReader.java

```diff
@@ -0,0 +1,35 @@
+package kr.co.wordle.support;
+
+import kr.co.wordle.domain.provider.AnswerProvider;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+
+import static kr.co.wordle.config.WordleGameConfig.WORDS_FILE_PATH;
+
+public class WordFileReader {
+
+    private WordFileReader() {
+    }
+
+    public static List<String> readWordsInFile() {
+        URL resource = AnswerProvider.class.getClassLoader().getResource(WORDS_FILE_PATH);
```

Comment:
다른 클래스를 참조하고 있어서 수정하는 게 좋을 것 같아요!
```suggestion
        URL resource = WordFileReader.class.getClassLoader().getResource(WORDS_FILE_PATH);
```

Reply:
리팩토링하다가 빠뜨렸네요 🥹 짚어주셔서 감사합니다 ~!